catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-common/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/common/TestFeatureMetacardType.java

```diff
@@ -463,7 +473,9 @@ public void testFeatureMetacardTypeFindBasicMetacardAttributes() {
                 NON_QUERYABLE_PROPS,
                 Wfs10Constants.GML_NAMESPACE);
 
-        for (AttributeDescriptor ad : BasicTypes.BASIC_METACARD.getAttributeDescriptors()) {
+        Set<AttributeDescriptor> descriptors = initDescriptors();
+
+        for (AttributeDescriptor ad : descriptors) {
             assertBasicAttributeDescriptor(fmt, ad.getName(), ad.getType());
```

Comment:
Do we expect the number of attribute descriptors in the `FeatureMetacardType` to be the same as what's in the `descriptors` set? If so we probably should have an assertion for that too.